tailwind.config.ts

```diff
@@ -62,44 +64,19 @@ export default {
         },
       ],
     },
-    spacing: {
-      px: "1px",
-      0: "0px",
-      0.5: "2px",
-      1: "4px",
-      1.5: "6px",
-      2: "8px",
-      2.5: "10px",
-      3: "12px",
-      3.5: "14px",
-      4: "16px",
-      5: "20px",
-      6: "24px",
-      7: "28px",
-      8: "32px",
-      9: "36px",
-      10: "40px",
-      11: "44px",
-      12: "48px",
-      14: "56px",
-      16: "64px",
-      20: "80px",
-      24: "96px",
-      28: "112px",
-      32: "128px",
-      36: "144px",
-      40: "160px",
-      44: "176px",
-      48: "192px",
-      52: "208px",
-      56: "224px",
-      60: "240px",
-      64: "256px",
-      72: "288px",
-      80: "320px",
-      96: "384px",
-    },
+    // use pixel scale by default
+    spacing: absoluteSpacing,
     extend: {
+      ...getRelativeSpacingOverride([
+        "flexBasis",
+        "height",
+        "maxHeight",
+        "maxWidth",
+        "minHeight",
+        "minWidth",
+        "textIndent",
+        "width",
+      ]),
```

Comment:
This is an incredibly minor nitpick and I'm absolutely not blocking the PR for this, but I'm worried that this is not actually typed? These keys might change in a future Tailwind update (highly unlikely) and we wouldn't notice.

What's the advantage of this over:

```ts
{
  flexBasis: relativeSpacing,
  height: relativeSpacing,
  maxHeight: relativeSpacing
  // ...
}
```

I mean, other than the obnoxious repetition 😆

Reply:
it is typed, it only allows keys from the Tailwind theme object. At the same time, I agree, let's make it simple